test/Microsoft.SqlTools.ServiceLayer.UnitTests/Profiler/ProfilerServiceTests.cs

```diff
@@ -58,11 +58,14 @@ public class ProfilerServiceTests
             await profilerService.HandleStartProfilingRequest(requestParams, requestContext.Object);
 
             // wait a bit for profile sessions to be polled
-            Thread.Sleep(500);
+            Thread.Sleep(1000);
```

Comment:
See my comment below for more about sleeping in tests... how do we know that 1000 is really enough if 500 sometimes isn't?